### PR TITLE
Fix xy coordinate data extract from cyme converter

### LINKED ISSUE #46: Extract XY coordinate data from CYME converter
## Background

It’s difficult to plot feeder data on a map or canvas without coordinate data. CYME database contains coordinate data. This data would be useful for plotting a feeder in a map view.  This data does not need to be the form of latitude and longitude, just X and Y data directly extract directly from CYME is enough.

## Proposed Solution

Extract XY from CYME and save it as a separate CSV file? This is the most straightforward way without modification to GLD itself. The attachment “Glow_NodeCoordinates.csv” is an example of file name and format that we are currently using in GLOW.

node,x,y
load_1,-119.0022,35.38528
load_2,-119.0022,35.38577
node_3,-119.0022,35.3846

## Alternatives

Alternative 1, since this is not latitude and longitude, it may need to add new parameters in GLD object. For example, “coordinate_x”, “coordinate_y” parameters.

Alternative 2, extract XY coordinate data and use it as value in latitude and longitude.

## Example file "Glow_NodeCoordinates.csv"
[Glow_NodeCoordinates.csv](https://github.com/slacgismo/gridlabd/files/8006825/Glow_NodeCoordinates.csv)

### PR DESCRIPTION
**Pull request by MitchellAV**
This commit fixes #46

### CODE REVIEW

**Comment by MitchellAV** [comment]
@dchassin I'll take a look. I think I know what happened

File: converters/autotest/test_mdb2glm_xy_coordinates.glm
Comment (dchassin) [inline converters/autotest/test_mdb2glm_xy_coordinates.glm:1]: Recommend not testing for `mdb-schema` missing anymore. 
1. `apt install mdbtools` doesn't always install it in the same folder on different platforms.
2. mdbtools should always be installed anyway.

File: converters/autotest/test_mdbtable2glm.glm
Comment (dchassin) [inline converters/autotest/test_mdbtable2glm.glm:1]: See above comment about testing for mdb-schema.

File: converters/autotest/test_mdbtable2glmobject.glm
Comment (dchassin) [inline converters/autotest/test_mdbtable2glmobject.glm:1]: See above comment about testing for mdc-schema.

**Review by dchassin — CHANGES_REQUESTED** [review]
See comments on autotests: remove check for mdb-schema. This should no longer be needed and it doesn't work on all platforms anyway.

**Review by dchassin — APPROVED** [review]

**Pull request merged**